antarest/launcher/service.py

```diff
@@ -603,74 +601,28 @@ def download_output(self, job_id: str) -> FileDownloadTaskDTO:
             return self.output_service.export_output(job_result.study_id, job_result.output_id)
         raise JobNotFound()
 
-    def get_load(self) -> LauncherLoadDTO:
+    def get_load(self, launcher_id: Optional[str]) -> LauncherLoadDTO:
         """
-        Get the load of the SLURM cluster or the local machine.
+        Get the load of the specified launcher.
         """
-        # SLURM load calculation
-        if self.config.launcher.default == "slurm":
-            if slurm_config := self.config.launcher.slurm:
-                ssh_config = SSHConfigDTO(
-                    config_path=Path(),
-                    username=slurm_config.username,
-                    hostname=slurm_config.hostname,
-                    port=slurm_config.port,
-                    private_key_file=slurm_config.private_key_file,
-                    key_password=slurm_config.key_password,
-                    password=slurm_config.password,
-                )
-                partition = slurm_config.partition
-                allocated_cpus, cluster_load, queued_jobs = calculates_slurm_load(ssh_config, partition)
-                args = {
-                    "allocatedCpuRate": allocated_cpus,
-                    "clusterLoadRate": cluster_load,
-                    "nbQueuedJobs": queued_jobs,
-                    "launcherStatus": "SUCCESS",
-                }
-                return LauncherLoadDTO(**args)
-            else:
-                raise KeyError("Default launcher is slurm but it is not registered in the config file")
-
-        # local load calculation
-        local_used_cpus = sum(
-            LauncherParametersDTO.from_launcher_params(job.launcher_params).nb_cpu or 1
-            for job in self.job_result_repository.get_running()
-        )
 
-        # The cluster load is approximated by the percentage of used CPUs.
-        cluster_load_approx = min(100.0, 100 * local_used_cpus / (os.cpu_count() or 1))
+        return self.config.launcher.get_load(self.job_result_repository.get_running(), launcher_id)
```

Comment:
In relation with previous comment: here we should ask the the launcher implementation instead:
```python
return self.launchers[cluster_id].get_load()
```
we don't need to provide the launcher with more information such as running jobs, since he knows itself which jobs are running